### ISSUE TITLE
[Spell][Mage] Blink 

### ISSUE THREAD

**Issue by killablink** [Atlantiss/NetherwingBugtracker#3401]
**Description**: Just a few bugs to clean with blink, but it's a lot better than what it was before

1st) https://gfycat.com/DownrightEnviousBasil
There is currently no maximum range to blink back to the ground. The max distance to blink to the ground should be around 30-40 yards ( To be honest I don't know how to measure it but if I could show you ingame I could give you a pretty reliable feel of when you should be able to blink to the ground from the air)  (If I were to give a visual estimation of where I feel blink from air to ground should stop working it would be around this height on this gif https://gfycat.com/RealisticUncommonElephantseal)

2) https://gfycat.com/MammothEquatorialBlacklemur)
 Can't blink to the ground directly from the mount (Probably because of the minimum falling time required in the way you programmed it I believe

3) https://streamable.com/1e3c8
   https://gfycat.com/WarpedUnsteadyFoxhound
Sometimes still able to blink from one platform to another, which should be impossible.

**Server Revision**: 2431

**Comment by Tenoutoften** [comment]
This video looks the most similar
https://www.youtube.com/watch?v=eiC7fX07cGE&feature=youtu.be&t=499

I don't know what is happening with the archimonde videos where the mage is not really blinking forwards
https://www.youtube.com/watch?v=0Tr4vjhawQc&feature=youtu.be&t=429
https://www.youtube.com/watch?v=0Tr4vjhawQc&feature=youtu.be&t=185

**Comment by Brasso** [comment]
Platform blinking does not work in Blade's Edge arena, I don't believe it's exploitable and just a minor issue. The only real issue that should be fixed before launch should be the instant blink to the ground above 40 yards. 

https://imgur.com/a/knpwMvJ

**Comment by somnu** [comment]
What about blinking up ramps, has anyone tested any ramps? I tested the small ones in shattarth (the ones you use to get up the bridges to exit towards terokkar) just briefly and it seems like it's not working properly. If anyone could be so kind as to test and record.

**Comment by Buildya** [comment]
> What about blinking up ramps, has anyone tested any ramps? I tested the small ones in shattarth (the ones you use to get up the bridges to exit towards terokkar) 

tested it, works for me

**Comment by somnu** [comment]
This is what happens for me:

https://cdn-b-east.streamable.com/video/mp4/o75sq.mp4?token=cEMVsScQl-eE1Lh1Cbsc-Q&expires=1544719952

**Comment by Brasso** [comment]
I think there's something wonky with Shattrath and not blink. Blinking in very steep slopes works well in other places.

https://imgur.com/a/84UPDAC

**Comment by killablink** [comment]
Here's a few exploitable blink in WSG in order of importance

https://gfycat.com/LateCrazyLemming
https://gfycat.com/WeightyInformalDarklingbeetle
https://gfycat.com/FluffyRectangularHuemul (Goes up to the alliance graveyard in the corner of the map)

This one is really hard to get, so I don't think it's too major if it's not fixed (I had to blink a good like 100-200 times to kinda understand how that one works)
https://gfycat.com/GleamingUnfinishedGraysquirrel

This corner in blade's edge is still bugged (more important cuz bugging in arena is a feelsbad)
https://gfycat.com/ThisImpoliteAbyssiniancat
https://gfycat.com/BetterNiceAustraliancattledog
It doesn't happen if you're right outside of the ramp
https://gfycat.com/UnrulyAbleDobermanpinscher

**Comment by Tenoutoften** [comment]
![](https://giant.gfycat.com/WeightyInformalDarklingbeetle.webm)

**Comment by Godmatik** [comment]
Rev 4020

All of these links are broken so it's difficult to properly analyze and test this bug report from 6 years ago. I imagine a lot of these exploits reported were already fixed, as I would've heard especially about the BG/arena related ones this past year on Netherwing.

- "There is currently no maximum range to blink back to the ground. The max distance to blink to the ground should be around 30-40 yards"

After testing and jumping from some high distances, there definitely is a max distance implemented

- "Also, when you blink back to the ground you should not blink right under where you were falling, but in front of it and gain distance"

Works fine for me - I properly blink forward and gain distance

- "Can't blink to the ground directly from the mount (Probably because of the minimum falling time required in the way you programmed it I believe"

I confirm that this is a bug, if you try to blink while mounted, it will do so mid-air only and you won't be put on the ground.

- "Sometimes still able to blink from one platform to another, which should be impossible."

Since all of the links are broken, I can not verify this.

**So to summarize - for now - just need to fix the issue of when using blink ability on a flying mount, not being teleported to the ground as you should be** - also it is slightly possible this is intended, but I am not too certain
